src/typings/transferWebhooks/transferDataTracking.ts

```diff
@@ -30,12 +30,7 @@ export type TransferDataTracking = ConfirmationTrackingData | EstimationTracking
 */
 export class TransferDataTrackingClass {
     
-    static readonly discriminator: string | undefined = "type";
+    static readonly discriminator: string = "type";
 
-    static readonly mapping: {[index: string]: string} | undefined = {
-        "confirmation": "ConfirmationTrackingData",
```

Comment:
Is this correct?

Reply:
We did a manual revert before, now we do this again:

https://github.com/Adyen/adyen-node-api-library/commit/9e9cdb7842dcb8bd512b03baae6a568ed726f64d